### DISCUSSION TITLE
Questions about licensing

### DISCUSSION THREAD

**Discussion by JasonTysonParker** [jgraph/docker-drawio#135] [Q&A]
Hello,

We are interested in using this repository, but are wondering how the licensing works.

Draw.io has two options listed for Confluence: Cloud and data centre.
![image](https://github.com/jgraph/docker-drawio/assets/150009849/eb93b86a-c09d-49c7-b614-87498d143f77)

We wonder how high the price for a JGraph container will be or how licensing will be handled.

Could someone help me with this topic? Thank you very much in advance.

Best Regards,

Jason Parker

**Comment by davidjgraph** [comment]
It's open source, the license is GPL 3.

**Comment by JasonTysonParker** [comment]
Thank you very much. @davidjgraph 

Best Regards,

Jason Parker
